plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -309,13 +309,21 @@ func (r *NodeAuthorizer) authorizeResourceSlice(nodeName string, attrs authorize
 		return authorizer.DecisionNoOpinion, "cannot authorize ResourceSlice subresources", nil
 	}
 
-	// allowed verbs: get, create, update, patch, delete
+	// allowed verbs: get, create, update, patch, delete, watch, list, deletecollection
 	verb := attrs.GetVerb()
 	switch verb {
 	case "get", "create", "update", "patch", "delete":
 		// Okay, but check individual object permission below.
-	case "watch", "list":
-		// Okay. The kubelet is trusted to use a filter for its own objects.
+	case "watch", "list", "deletecollection":
+		// Okay. The kubelet is trusted to use a filter for its own objects in watch and list.
```

Comment:
I'll update https://github.com/kubernetes/kubernetes/pull/125571 to enforce the selector for this ... what is the exact field selector kubelets will use to list / watch / deletecollection resource slices? `nodeName=$name`?

Reply:
Currently yes, but it will become `spec.nodeName` once https://github.com/kubernetes/kubernetes/pull/125488 merges because that moves top-level fields into a `ResourceSliceSpec`.

Reply:
I updated https://github.com/kubernetes/kubernetes/pull/125571 to enforce a `nodeName=$name` selector... will tweak that to spec.nodeName=$name if https://github.com/kubernetes/kubernetes/pull/125488 beats it in